client/src/components/Landing.js

```diff
@@ -1,17 +1,24 @@
 import React from 'react'
 import { useNavigate } from "react-router-dom"
 import { useUser } from "../contexts/UserProvider"
+import { useState } from 'react';
+import { Alert } from 'react-bootstrap';
 
 function Landing() {
 	const navigate = useNavigate();
 	const { user, setUser } = useUser();
+	const [success, setSuccess] = useState(true)
```

Comment:
Be verbose, what does `success` signify here? Maybe do something like `loginSuccess`

Reply:
made it to the login success